readthedocs/projects/models.py

```diff
@@ -1446,6 +1446,13 @@ class Domain(TimeStampedModel, models.Model):
         blank=True,
     )
 
+    active = models.BooleanField(
+        _('Active'),
```

Comment:
What's the definition of an "active domain"? Does it have at least one hit request? Does it have SSL configured? Does it have the DNS records in place? Everything?

Reply:
There is a PR in -ext that explains this.

Reply:
Right, but it would be good to not require reading a PR in -ext to understand what this field means. That's my point.

I think this field should have a better name and be self-explanatory.

Reply:
What other name you suggest? The meaning of active is different from .org and .com.

Reply:
What it means on .org? and .com?

Reply:
The pr in ext has all this information (`#362`)

Reply:
We should figure out how to move forward here. @stsewd can you explain more?

Reply:
A domain is marked as active if it's ready (is_ready in `#362`). For .org it must have a domainssl object, a valid ssl status and the cname record; for .com is the same plus `domain.https = True` and the ops email must have been sent. The field is only used in -ext.

Reply:
Makes sense. We should put this into a code comment next to the field, explaining what it means on each case and making a reference to `Provider.is_ready` so we know where to find these differences between .com/.org